src/components/AttemptsCounter.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+
+interface Props {
+  retryLimit: number
+  retriesAllowed: number
+}
+
+export const AttemptsCounter: React.FC<Props> = ({ retryLimit, retriesAllowed }) => (
+    <div className="os-flex os-justify-content-end">
+        {retryLimit === 0
+          ? (
+                <p className="os-attempts-text">
+                    Attempts left: Unlimited
+                </p>
+            )
+          : (
+                <p className="os-attempts-text">
+                    Attempts left: {retryLimit - retriesAllowed + 1}/
+                    {retryLimit + 1}
+                </p>
+            )}
+    </div>
+)
```

Comment:
Minor / white space inconsistency: The indentation here is 4 spaces vs 2.

Reply:
Fixed.

![image](https://github.com/openstax/k12-apps-raise/assets/37000585/4f076910-9ae0-4a82-b629-961e8120cb94)

Reply:
Updated per this [commit](https://github.com/openstax/k12-apps-raise/pull/89/commits/040f8931d9c2d6451b0d23a1920bccde6a4843cd).